packages/fiori/src/UserSettingsAccountView.ts

```diff
@@ -0,0 +1,104 @@
+import UserSettingsView from "./UserSettingsView.js";
+import UserSettingsAccountViewTemplate from "./UserSettingsAccountViewTemplate.js";
+import {
+	customElement, slot, eventStrict as event, property,
+} from "@ui5/webcomponents-base/dist/decorators.js";
+import jsxRenderer from "@ui5/webcomponents-base/dist/renderer/JsxRenderer.js";
+import type I18nBundle from "@ui5/webcomponents-base/dist/i18nBundle.js";
+import UserSettingViewCss from "./generated/themes/UserSettingsView.css.js";
+import UserSettingsAccountViewCss from "./generated/themes/UserSettingsAccountView.css.js";
+import type UserMenuAccount from "./UserMenuAccount.js";
+
+// Texts
+import {
+	USER_SETTINGS_ACCOUNT_EDIT_AVATAR_TXT,
+	USER_SETTINGS_ACCOUNT_MANAGE_ACCOUNT_BUTTON_TXT,
+} from "./generated/i18n/i18n-defaults.js";
+import i18n from "@ui5/webcomponents-base/dist/decorators/i18n.js";
+
+@customElement({
+	tag: "ui5-user-settings-account-view",
+	renderer: jsxRenderer,
+	template: UserSettingsAccountViewTemplate,
+	styles: [UserSettingViewCss, UserSettingsAccountViewCss],
+})
+/**
+ * Fired when the "Edit Accounts" button is selected.
```

Comment:
Maybe it's better to use the backticks to format the button label: Fired when the `Edit Accounts` button is selected.